### DISCUSSION TITLE
Logitech G Hub

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#36] [General]
# Logitech G Hub

Check if Logitech G Hub is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/3a773c77-9d3d-44b4-b178-92a1884f4ace

**Comment by xmlnsgit** [comment]
Just installed and it is emulated

**Reply by OpenSource03** [reply]
We have just tested it and it indeed works under emulation. I’ve updated the status. It may take up to two hours for the changed to propagate.

**Comment by Morris-Chen** [comment]
Althrough it is emulated and can be execluted, but the function of assign custom action to different buttons are not working as it fails to capture connected mouse is logitech mouse thus no device is shown in the device tab.
so I wound't say it is okay, it is only able to be emulated, but has zero functionality.
